website/docs/r/route_entry.html.markdown

```diff
@@ -60,9 +114,9 @@ The following attributes are exported:
 
 ## Import
 
-Router entry can be imported using the id, e.g.
+Router entry can be imported using the id, e.g(formatted as<route_table_id:router_id:destination_cidrblock:nexthop_type:nexthop_id>).
```

Comment:
Please use the correct id format. I think its id should be foramted as `<route_table_id>:<route_entry_id>`